client/src/js/auth-helpers.js

```diff
@@ -0,0 +1,103 @@
+import { h, Component, render } from 'preact';
+import Cookies from 'universal-cookie';
+import { route } from 'preact-router';
+import decode from 'jwt-decode';
+import { Menu } from 'preact-material-components';
+import {API_SERVER} from "../../config";
+import axios from "axios/index";
+
+const tokenCookie = 'token';
```

Comment:
obfuscate for security reasons: we should consider renaming this and pull from a secrets.json file so its not obvious to outside that this the name for our authentication token. *(low-priority - but something to think about for production release)*